### PR TITLE
Added spinnaker + binary authorization tutorial

### PR DESCRIPTION
**Pull request by damadei-google**
Tutorial on creating attestations by integrating Cloud Build, Spinnaker, Cloud KMS and Binary Authorization to securely deploy Kubernetes applications

### CODE REVIEW

File: tutorials/spinnaker-binary-auth/index.md [hunk 1]
```diff
@@ -0,0 +1,702 @@
...
+description: Creating attestations by integrating Cloud Build, Spinnaker, Cloud KMS and Binary Authorization to securely deploy Kubernetes applications
+author: damadei
+tags: Kubernetes, Spinnaker, CI/CD, DevOps, Binary Authorization, Cloud KMS, Cloud Build
+---
+
+Daniel Amadei | Customer Engineer | Google
+
+# Introduction
+
+This solution exemplifies how to perform continuous integration and continuous delivery to a Kubernetes cluster using Cloud Build for building our images, Spinnaker for Continuous Deployment and Binary Authorization with keys hosted inside Cloud Key Management Service (Cloud KMS) for protecting and attesting the procedence of our images being deployed to Kubernetes.
```
Comment (siliconmoo) [inline tutorials/spinnaker-binary-auth/index.md:12]:
```suggestion
This solution shows you how to use a continuous integration (CI) and continuous delivery (CD) pipeline to securely deploy to a Kubernetes cluster. It uses Cloud Build to build images, Spinnaker for continuous delivery and Binary Authorization for deploy-time, attestation-based security, and Cloud Key Management Service (Cloud KMS) to manage keys used by Binary Authorization.  Using these Google Cloud products, you can  manage the provenance of container images deployed to Kubernetes. 
```

File: tutorials/spinnaker-binary-auth/index.md [hunk 2]
```diff
@@ -0,0 +1,702 @@
...
+tags: Kubernetes, Spinnaker, CI/CD, DevOps, Binary Authorization, Cloud KMS, Cloud Build
+---
+
+Daniel Amadei | Customer Engineer | Google
+
+# Introduction
+
+This solution exemplifies how to perform continuous integration and continuous delivery to a Kubernetes cluster using Cloud Build for building our images, Spinnaker for Continuous Deployment and Binary Authorization with keys hosted inside Cloud Key Management Service (Cloud KMS) for protecting and attesting the procedence of our images being deployed to Kubernetes. 
+
+We will start by setting up all our environment and then building the solution step by step. We hope you enjoy the journey.
```
Comment (siliconmoo) [inline tutorials/spinnaker-binary-auth/index.md:14]: [A couple general thoughts:
1. First, this is my first review! So let's make sure everything is working with the review cycle, and then I'm happy to review further.

2. Generally speaking in this kind of writing, it's fine to drop the "we"/"we will" and just tell them what to do and using the second person singular "you" vs plural "we". Also, present tense is fine.  Putting both pieces of feedback together the above sentence becomes: "Start by setting up your environment."
]

**Comment by ToddKopriva** [comment]
@siliconmoo , thanks for helping out with the review.

I'll be doing a copy edit and a production pass (dealing with formatting and such), so you can forego that part of the review if you'd rather.

File: tutorials/spinnaker-binary-auth/index.md (hunk 2 above)
Comment (damadei-google) [reply tutorials/spinnaker-binary-auth/index.md:14]: Great, will follow that!

**Comment by ToddKopriva** [comment]
All contributors are Google employees, so I set the CLA "yes" tag.

**Pull request merged**